### PR TITLE
Marketing dev docs review  

### PR DESCRIPTION
**Pull request by Otsar-Raikou**
we briefly reviewed the documentation, fixing spelling errors and the sidebar as well as centering some images and editing pages, we consider this as at least a first version of cleanup and fixing to the documentation as it is an ongoing process.

### CODE REVIEW

File: docs/docs/icicle/arch_overview.md [hunk 1]
```diff
@@ -1,34 +1,28 @@
 # Architecture Overview
 
-## Introduction
 
-ICICLE V3 is designed with flexibility and extensibility in mind, offering a robust framework that supports multiple compute backends and accommodates various cryptographic needs. This section provides an overview of ICICLE's architecture, highlighting its open and closed components, multi-device support, and extensibility.
+ICICLE v3 is designed with flexibility and extensibility in mind, offering a robust framework that supports multiple compute backends and accommodates various cryptographic needs. This section provides an overview of ICICLE's architecture, highlighting its open and closed components, multi-device support, and extensibility.
 
-## Open Frontend and CPU Backend
+### Open Frontend and CPU Backend
```
Comment (jeremyfelder) [inline docs/docs/icicle/arch_overview.md:6]: Do you want this to be a H3 instead of an H2?

Comment (Otsar-Raikou) [reply docs/docs/icicle/arch_overview.md:6]: its on purpose - theres no page corresponding so it seemed confusing

Comment (jeremyfelder) [inline docs/docs/icicle/arch_overview.md:6]: I would remove the "Open" from the header here if we aren't having "Closed" below on CUDA backend.

File: docs/docs/icicle/build_from_source.md [hunk 2]
```diff
@@ -175,6 +175,4 @@ Make sure to install icicle libs when installing a library/application that depe
 
 ### Go: Build, Test, and Install (TODO)
 
-## Install cuda backend
-
-[Install CUDA Backend (and License)](./install_cuda_backend.md#installation)
\ No newline at end of file
+## [Install CUDA backend & License](./install_cuda_backend.md#installation)
```
Comment (jeremyfelder) [inline docs/docs/icicle/build_from_source.md:178]: A bit strange to have a Header for a section that is a link and nothing in the section

File: docs/docs/icicle/install_cuda_backend.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ The CUDA backend is a closed-source component that requires a license. [To insta
 ### Licensing
 
 :::note
-Currently, the CUDA backend is free to use via Ingonyama’s icicle-cuda-backend-license server. By default, the CUDA backend will attempt to access this server. For more details, please contact support@ingonyama.com.
+Currently, the CUDA backend is free to use via Ingonyama’s ICICLE-CUDA-backend-license server. By default, the CUDA backend will attempt to access this server. For more details, please contact support@ingonyama.com.
```
Comment (jeremyfelder) [inline docs/docs/icicle/install_cuda_backend.md:15]: No need to complicate the naming of this
```suggestion
Currently, the CUDA backend is free to use via Ingonyama’s backend license server. By default, the CUDA backend will attempt to access this server. For more details, please contact support@ingonyama.com.
```

File: docs/docs/icicle/overview.md [hunk 4]
```diff
@@ -5,8 +5,14 @@ title: ICICLE Overview
...
 # ICICLE Overview
 
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/794c306a-b5fc-425c-8b1b-60b87aa12200" alt="Untitled design (29)" width="500"/>
+</p>
+
+
 ## What is ICICLE?
 
+
```
Comment (jeremyfelder) [inline docs/docs/icicle/overview.md:15]: No need for another blank line here

File: docs/docs/icicle/overview.md [hunk 5]
```diff
@@ -69,9 +75,9 @@ ICICLE is also well-suited for prototyping and developing small-scale projects.
 
 ## Get Started with ICICLE
 
-Explore the full capabilities of ICICLE by diving into the [Architecture](./arch_overview.md), [Getting Started Guide](./getting_started.md) and the [programmers guide](./programmers_guide/general.md) to learn how to integrate, deploy, and extend ICICLE across different backends.
+Explore the full capabilities of ICICLE by diving into the [Architecture](./arch_overview.md), [Getting Started Guide](./getting_started.md) and the [Programmers guide](./programmers_guide/general.md) to learn how to integrate, deploy, and extend ICICLE across different backends.
```
Comment (jeremyfelder) [inline docs/docs/icicle/overview.md:78]:
```suggestion
Explore the full capabilities of ICICLE by diving into the [Architecture](./arch_overview.md), [Getting Started Guide](./getting_started.md) and the [Programmer's Guide](./programmers_guide/general.md) to learn how to integrate, deploy, and extend ICICLE across different backends.
```

File: docs/docs/icicle/overview.md [hunk 6]
```diff
@@ -69,9 +75,9 @@ ICICLE is also well-suited for prototyping and developing small-scale projects.
...
 
-If you have any questions or need support, feel free to reach out on [Discord] or [GitHub](https://github.com/ingonyama-zk). We're here to help you accelerate your ZK development with ICICLE.
+If you have any questions or need support, feel free to reach out on [Discord], [GitHub](https://github.com/ingonyama-zk) or [support@ingonyama.com](support@ingonyama.com). We're here to help you accelerate your ZK development with ICICLE.
```
Comment (jeremyfelder) [inline docs/docs/icicle/overview.md:80]: No need to explicitly link to an email
```suggestion
If you have any questions or need support, feel free to reach out on [Discord], [GitHub](https://github.com/ingonyama-zk) or <support@ingonyama.com>. We're here to help you accelerate your ZK development with ICICLE.
```

File: docs/docs/icicle/build_from_source.md (hunk 2 above)
Comment (Otsar-Raikou) [reply docs/docs/icicle/build_from_source.md]: fixed

File: docs/docs/icicle/arch_overview.md (hunk 1 above)
Comment (Otsar-Raikou) [reply docs/docs/icicle/arch_overview.md]: fixed

File: docs/docs/icicle/overview.md (hunk 4 above)
Comment (Otsar-Raikou) [reply docs/docs/icicle/overview.md:15]: theres a static bade - unsure what u mean

**Review by yshekel — APPROVED** [review]

**Review by jeremyfelder — APPROVED** [review]

**Pull request merged**